libraries/lambda_handlers/convert_handler.py

```diff
@@ -36,4 +42,22 @@ def _handle(self, event, context):
         converter = self.converter_class(source=source, resource=resource, cdn_file=cdn_file, options=options)
         results = converter.run()
         converter.close()  # do cleanup after run
+        if callback is not None:
+            self.callback_payload = json.loads(json.dumps(self.data))  # clone so we can modify
+            self.callback_payload['results'] = results  # add results to payload and call back
```

Comment:
It's ok to just send back `job_id` and `results`... it doesn't need the whole job payload.